api/src/auth/drivers/openid.ts

```diff
@@ -204,9 +204,15 @@ export class OpenIDAuthDriver extends LocalAuthDriver {
 
 		let role = this.config['defaultRoleId'];
 		const groupClaimName: string = this.config['groupClaimName'] ?? 'groups';
-		const groups = userInfo[groupClaimName];
+		let groups: string[] = [];
 
-		if (Array.isArray(groups)) {
+		if (userInfo[groupClaimName]) {
+			groups = Array.isArray(userInfo[groupClaimName])
+				? userInfo[groupClaimName]
+				: [userInfo[groupClaimName]].filter(Boolean);
+		}
```

Comment:
I would suggest we use our `toArray` util here
```suggestion
		if (userInfo[groupClaimName]) {
			groups = toArray(userInfo[groupClaimName])
		}
```